ui-tests/src/test/java/org/openmrs/reference/EndVisitTest.java

```diff
@@ -33,20 +32,19 @@ public void setUp() throws Exception {
     }
 
     @Test
-    @Ignore
     @Category(BuildTests.class)
-    public void EndVisitTest() throws Exception {
+    public void endVisitTest() {
         FindPatientPage findPatientPage = homePage.goToFindPatientRecord();
         findPatientPage.enterPatient(patient.identifier);
+        findPatientPage.waitForPageToLoad();
         ClinicianFacingPatientDashboardPage clinicianFacingPatientDashboardPage = findPatientPage.clickOnFirstPatient();
         PatientVisitsDashboardPage patientVisitsDashboardPage = clinicianFacingPatientDashboardPage.goToRecentVisits();
         patientVisitsDashboardPage = patientVisitsDashboardPage.endVisit();
-        assertNull(patientVisitsDashboardPage.getActiveVisit());
+        assertTrue(patientVisitsDashboardPage.containsText("No active visit"));
```

Comment:
thanks @kdaud  what happens in instances when we the text "No active visit"  is in another language

Reply:
> what happens in instances when we the text

Was this a typo ? I can't get the point clear.

Reply:
> what happens in instances when the text "No active visit" is in another language

Good question @HerbertYiga. AFAICT in that instance the test `MUST` fail. And for quality assurance such instances should be catered for as well.

Reply:
BTW is our system translated and used in other language than English? If so do we have a squad that does this ?